src/js/common/components/CampaignShare/ShareByCopyLink.jsx

```diff
@@ -96,6 +97,23 @@ class ShareByCopyLink extends Component {
     if (this.props.onClickFunction) {
       this.props.onClickFunction();
     }
+    // adding dataLayer for "Copy link" on Share Ballot
+    const currentPathname = window.location.pathname;
+
+    const dataLayerObject = {
+      event: 'ShareBallotCopyLinkClick',
+      shareDetails: {
```

Comment:
I am adding shareDetails to the Property Name tab of the [Solution Design spreadsheet.](https://docs.google.com/spreadsheets/d/1KO4_AOmddKvIpA79XrZdaVBdRkcP1HitqrvQkCo2A34/edit?pli=1&gid=1828322090#gid=1828322090)